scripts/compute-tags.sh

```diff
@@ -36,4 +37,5 @@ fi
 
 IMAGE_TAGS="${IMAGE_TAGS} --tag docker.io/${SNAPSHOT_TAG} --tag quay.io/${SNAPSHOT_TAG}"
 
+#shellcheck disable=SC2086
```

Comment:
`echo "${IMAGE_TAGS}"` - why wouldn't this work? We are not passing the string as argument to another script, we just print it, and it should work fine with spaces.